internal/plugin/connectors/http/generic/oauthprotocol/protocol.go

```diff
@@ -0,0 +1,214 @@
+package oauth1protocol
```

Comment:
I think maybe just `oauth/v1/protocol.go` might be a better layout of this. Thoughts?